### ISSUE TITLE
reduce lsf resource requirement for RNASeq pipeline.

### ISSUE THREAD

**Issue by gatoravi** [genome/gms#158]
The current lsf resource requirement for the "RnaSeq Cufflinks Expression" step in the RNASeq pipeline is set to mem>64 GB. While this might be pretty fool-proof and work well inside our nodes at TGI it might not be ideal for the sGMS which might be installed on comparatively puny machines. 32 GB might be sufficient for the Cufflinks step ? 
@jasonwalker80 might know more about this and where this code change could happen.

**Comment by tmooney** [comment]
FWIW we sometimes have failures where 64 GB is insufficient to run Cufflinks, and we have to make requests of 150 or even 300 GB for it to complete.

**Comment by jasonwalker80** [comment]
Tom is correct about the Cufflinks resource demands.  If the sGMS RNA-seq processing profile were to run in 'reference only' mode and the parameters for --max-bundle-length and --max-bundle-frags were returned to the default values, I think 32GB would be sufficient.  Internally, we have determined we want de novo transcript mode (reference-guided mode is being phased out) and values of 10 million for the two Cufflinks parameters.

**Comment by malachig** [comment]
@gatoravi, lets try this with 32 GB.

**Comment by malachig** [comment]
@jasonwalker80 or @tmooney can you remind us how this gets configured?  Hard coded in the code?  Where specifically?

**Comment by sakoht** [comment]
There is an rusage method in Genome::InstrumentData::AlignmentResult::Tophat.

Is this the place?

**Comment by tmooney** [comment]
You can find it in [`Genome::Model::RnaSeq::Command::Expression::Cufflinks`](https://github.com/genome/genome/blob/67cfd1cc1ce23d5c0de01ae43a493d61e262025c/lib/perl/Genome/Model/RnaSeq/Command/Expression/Cufflinks.pm#L10-L12).

**Comment by gatoravi** [comment]
Looks like this processing-profile is 'reference-only' and default parameters are used for ' --max-bundle-length' and '--max-bundle-frags', 

```
ubuntu@ip-172-31-19-86 /opt/gms/RGTBV26/sw/workflow/lib/Workflow (gms-pub-2015.02.22)> genome processing-profile describe 2762841
'processing_profiles' may require verification...
Resolving parameter 'processing_profiles' from command argument '2762841'... found 1
Processing Profile: October 2012 Default Ovation V2 RNA-seq <ID: 2762841>
type_name: rna seq
annotation_reference_transcripts_mode: reference only
bedtools_version: 2.14.3
bowtie_version: 2.0.0-beta7
calculate_error_rate: 1
deduplication_handler: <NULL>
digital_expression_detection_strategy: <NULL>
dna_type: cdna
error_rate_version: 0.9
expression_name: cufflinks
expression_params: -p 4
expression_version: 2.0.2
fastqc_version: 0.10.0
fusion_detector: <NULL>
fusion_detector_params: <NULL>
fusion_detector_version: <NULL>
mask_reference_transcripts: rRNA_MT
picard_strand_specificity: NONE
picard_version: 1.52
read_aligner_name: tophat
read_aligner_params: -p 4 --bowtie-version=2.0.0-beta7
read_aligner_version: 2.0.4
read_trimmer_name: far
read_trimmer_params: --adapter CTTTGTGTTTGA --trim-end left --adaptive-overlap yes --format fastq --write-lengthdist yes --nr-threads 4 --min-overlap 7 --max-uncalled 150 --min-readlength 25
read_trimmer_version: 2.17
samstat_version: 1.08
samtools_version: r963
sequencing_platform: solexa
transcript_coverage_merge_annotation_features: yes
transcriptome_coverage_annotation_file_basenames: none
transcriptome_coverage_mask_reference_transcripts: MT
```

So it might potentially be safe to use 32GB as the lsf parameter. Not sure how big of a priority this is but obviously this will be another point of divergence from master. If this sounds ok I can add this commit to the SGMS branch of genome.

**Comment by brianpenghe** [comment]
Hi Gatoravi, can I ask where CTTTGTGTTTGA comes from? Your answer would help me a lot. Thanks

**Comment by malachig** [comment]
I believe this is the expected adapter sequence for the primer used in the linear amplification step of the Nugen Ovation RNA-seq library construction method.  This adapter is attached to the ends of cDNAs, used to drive a linear SPIA amplification (http://www.nugen.com/sites/default/files/DS_v4-Ovation_RNA-Seq_V2.pdf).  The cDNA is then fragmented, but the resulting fragments that correspond to the ends of these cDNA still contain the SPIA adapter.  In this processing profile we therefore use trimming to remove SPIA adapter sequence.

I can't find a reference for the sequence itself.  It may have been provided to us directly by the manufacturer of the kit.

**Comment by brianpenghe** [comment]
Yes I found other independent sources saying that this comes from NuGEN whose official website never released the actual sequence. Thank you very much for your answer!

**Comment by brianpenghe** [comment]
Actually, something to add: I found that NuGEN kit also has another 5' adapter GGTAACTTTGTGTTT
